<ui version="4.0" >
 <class>DesktopNotificationConfigWidget</class>
 <widget class="QWidget" name="DesktopNotificationConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="enabled" >
     <property name="title" >
      <string>Desktop Notification (via D-Bus)</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QCheckBox" name="useTimeout" >
          <property name="text" >
           <string>Timeout:</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="timeout" >
          <property name="specialValueText" >
           <string/>
          </property>
          <property name="suffix" >
           <string> s</string>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QCheckBox" name="useHints" >
          <property name="text" >
           <string>Position hint:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="xHint" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="suffix" >
           <string> px</string>
          </property>
          <property name="prefix" >
           <string>X: </string>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="yHint" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="suffix" >
           <string> px</string>
          </property>
          <property name="prefix" >
           <string>Y: </string>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="queueNotifications" >
        <property name="text" >
         <string>Queue unread notifications</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>useHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>xHint</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>92</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel" >
     <x>156</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useHints</sender>
   <signal>toggled(bool)</signal>
   <receiver>yHint</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>51</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel" >
     <x>262</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTimeout</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeout</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>58</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>128</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
